spec/unit/data

```diff
@@ -0,0 +1 @@
+../../test
```

Comment:
Also random extra crap in your tree ;o)

Reply:
Thanks. I always forget that ./kodev clean doesn't clean up _everything_. I'll make another commit after I edit the pattern matching above.

Reply:
Thanks, deleted.